File: src/classes/api/req_res/get-tags-by-target-id-response.ts
'use strict';

import { Tag } from '@/classes/datas/tag';
import { GkillAPIResponse } from '../gkill-api-response';


export class GetTagsByTargetIDResponse extends GkillAPIResponse {


    tags: Array<Tag>;

    constructor() {
        super()
        this.tags = new Array<Tag>()
    }


}

File: src/server/gkill/api/req_res/get_tags_by_target_id_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type GetTagsByTargetIDResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	Tags []reps.Tag `json:"tags"`
}
